### ISSUE TITLE
Getting accuracy as 0.00

### ISSUE THREAD

**Issue by Arvind-Kumar-Agrawal** [zhiqwang/image-captioning#19]
I am trying to train a model but always getting accuracy as 0.00 :)
My data folder ->
[data.zip](https://github.com/zhiqwang/crnn.pytorch/files/3268869/data.zip)
**Command used** : python ./main.py --dataset-root G:\11\crnn.pytorch-master\crnn.pytorch-master\data --arch densenet121 --alphabet G:\11\crnn.pytorch-master\crnn.pytorch-master\data\alphabet_decode_5990.txt --lr 5e-5 --optimizer rmsprop 
Training Log screenshot : 
![image](https://user-images.githubusercontent.com/28316665/59151585-42413880-8a53-11e9-8beb-29c592f43e4e.png)

Need help here, thanks in advance for same

**Comment by zhiqwang** [comment]
First choose `densenet_cifar` model, large learning rate, for example 0.001. Besides, I think this dataset is too small to train a model.
